components/link-social/css/index.scss

```diff
@@ -5,21 +5,24 @@
 
 .utrecht-link-social {
   --utrecht-icon-size: var(--utrecht-link-social-icon-size);
+  --utrecht-icon-color: var(--utrecht-link-social-color);
 
   align-items: center;
   background-color: var(--utrecht-link-social-background-color);
   border-color: var(--utrecht-link-social-border-color);
   border-radius: 50%;
   border-style: solid;
   border-width: var(--utrecht-link-social-border-width);
-  color: var(--utrecht-link-social-color);
```

Comment:
I would rather leave the `color` so a text a hypothetical text alternative to the icon automatically has enough contrast.